src/avfoundation.cs

```diff
@@ -10764,6 +11378,46 @@ interface AVSpeechSynthesizerDelegate {
 		void WillSpeakRangeOfSpeechString (AVSpeechSynthesizer synthesizer, NSRange characterRange, AVSpeechUtterance utterance);
 	}
 
+	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVAssetDownloadStorageManager
+	{
+		[Static]
+		[Export ("sharedDownloadStorageManager")]
+		AVAssetDownloadStorageManager SharedDownloadStorageManager { get; }
+
+		[Export ("setStorageManagementPolicy:forURL:")]
+		void SetStorageManagementPolicy (AVAssetDownloadStorageManagementPolicy storageManagementPolicy, NSUrl downloadStorageUrl);
+
+		[Export ("storageManagementPolicyForURL:")]
+		[return: NullAllowed]
+		AVAssetDownloadStorageManagementPolicy GetStorageManagementPolicy (NSUrl downloadStorageUrl);
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[BaseType (typeof(NSObject))]
+	[DisableDefaultCtor]
+	interface AVAssetDownloadStorageManagementPolicy : NSCopying, NSMutableCopying
+	{
+		[Export ("priority")]
+		AVAssetDownloadedAssetEvictionPriority Priority { get; }
+
+		[Export ("expirationDate", ArgumentSemantic.Copy)]
+		NSDate ExpirationDate { get; }
+	}
+
+	[NoWatch, NoTV, NoMac, iOS (11,0)]
+	[BaseType (typeof(AVAssetDownloadStorageManagementPolicy))]
+	interface AVMutableAssetDownloadStorageManagementPolicy
+	{
+		[Export ("priority")]
+		AVAssetDownloadedAssetEvictionPriority Priority { get; set; }
+
+		[Export ("expirationDate", ArgumentSemantic.Copy)]
+		NSDate ExpirationDate { get; [NotImplemented] set; }
```

Comment:
Why `[NotImplemented]`?

Reply:
Please look at @spouliot comment.

Reply:
Not sure which comment but `[NotImplemented]` is usually used this way for the *non*-mutable type. Since this one is named `AVMutableAssetDownloadStorageManagementPolicy` (Mutable) then it's not clear why it's used.